### PR TITLE
Iteration 7

### PR DESCRIPTION
**Pull request by c-brenn**
do your thing circle

### CODE REVIEW

**Review by laurmurclar — DISMISSED** [review]
** EDIT IGNORE, thought this was being added to a line**We should specify that he needs to be in the pathways directory & that there is no need to have the system running already.

File: doc/FEATURES_RELEASE_2.md [hunk 1]
```diff
@@ -53,22 +54,136 @@ must display useful error messages about such constructs to the user.
...
+
+## Specify a Delay - Complete
+
+### Description
+The system should provide a way to say, at any point in a sequential workflow,
+that the enactment of the pathway will pause/delay for a given time.
+
+Users can specify a delay using the following construct:
+
+`time { ...list of time values }`
```
Comment (laurmurclar) [inline doc/FEATURES_RELEASE_2.md]: We should  probably describe the options for time values explicitly

Comment (laurmurclar) [reply doc/FEATURES_RELEASE_2.md]: Listing the valid time ranges for each value is worth doing too. We should also mention that order is irrelevant.

Comment (c-brenn) [reply doc/FEATURES_RELEASE_2.md]: 👍 agreed, will add that to the other PR

**Review by laurmurclar — APPROVED** [review]
🙌

**Pull request merged**